src/app/pages/ArticlePage/experimentTopStories/helpers.tsx

```diff
@@ -2,34 +2,58 @@
 import { jsx } from '@emotion/react';
 import { OptimoBlock } from '#app/models/types/optimo';
 import { TopStoryItem } from '#app/pages/ArticlePage/PagePromoSections/TopStoriesSection/types';
+import { buildATIEventTrackUrl } from '#app/components/ATIAnalytics/atiUrl';
+import { Services, Environments } from '#app/models/types/global';
 import TopStoriesSection from '../PagePromoSections/TopStoriesSection';
 import styles from './index.styles';
 
+export const experimentName = 'topStoriesExperiment';
 export const experimentTopStoriesConfig = {
-  topStoriesExperiment: {
+  [experimentName]: {
     variants: {
       control: 50,
       show_at_halfway: 50,
     },
   },
 };
 
+const ARTICLE_LENGTH_THRESHOLD = 10;
 const enableExperimentTopStories = ({
```

Comment:
Good point. I think I was just struggling for a name since `shouldEnableExperimentTopStories` is returned in the object from 'getExperimentTopStories()`. Happy to accept any other suggestions.
https://github.com/bbc/simorgh/blob/39bcd024d5574228a1baf3415a915683bb35b2a6/src/app/pages/ArticlePage/experimentTopStories/helpers.tsx#L81